### DISCUSSION TITLE
How to use tf.obsidian to open a file in current pane

### DISCUSSION THREAD

**Discussion by LumenYoung** [SilentVoid13/Templater#721] [Help]
Hi, as the title suggest, I would like to know how to use the obsidian api (since I have no knowledge in JS so I can only do basic scripting but not reading the api file exposed from obsidian) to open a file with specific name.

All helps are appreciated, thanks!

**Comment by AB1908** [comment]
Where are you stuck with this? Did you find anything useful in the Obsidian API?
